release_notes/364-release-notes.adoc

```diff
@@ -0,0 +1,57 @@
+[id="364-release-notes"]
+= Red Hat Advanced Cluster Security for Kubernetes 3.64
+include::modules/common-attributes.adoc[]
+:context: 364-release-notes
+
+toc::[]
+
+{product-title} 3.64 includes feature enhancements, bug fixes, scale improvements, and other changes.
+
+//To upgrade to this release from a previous version, see the link:/docs/upgrade-stackrox/[Upgrade]section.
+
+*Release date:* August 11, 2021
+
+[id="new-features"]
+== New features
+
+* *ROX-7230*: You can now use deployment and namespace annotations to define where {product-title} sends the violation notifications when configuring your notifiers such as Slack, Microsoft Teams, Email, and others.
+* *ROX-7534*: With {product-title} Operator, you can now configure the enforcement behavior of the admission controller as part of the custom resource setting.
+* *ROX-7561*: {product-title} now supports kernel modules for Ubuntu 16.04 LTS with extended security maintenance (ESM).
+
+[id="important-bug-fixes"]
+== Important bug fixes
+
+* *ROX-6326*: Previously, users with a considerably large number of namespaces would receive the `sporadic 504` gateway errors when sending requests to the `/v1/namespaces/` endpoint. {product-title} includes the updated endpoint, which supports pagination to fix this issue.
+
+[id="important-system-changes"]
+== Important system changes
+
+* *ROX-6258*: {product-title} now pre-fixes the optional security context constraint name with `stackrox` to avoid global naming conflicts.
```

Comment:
```suggestion
* *ROX-6258*: {product-title} now prefixes the optional security context constraint name with `stackrox` to avoid global naming conflicts.
```